docs-web/src/main/webapp/src/locale/en.json

```diff
@@ -341,8 +341,9 @@
         "type": "Step type",
         "type_approve": "Approve",
         "type_validate": "Validate",
+        "type_resumereview": "Resume Review",
         "target": "Assigned to",
-        "target_help": "<strong>Approve:</strong> Accept or reject the review<br/><strong>Validate:</strong> Review and continue the workflow",
+        "target_help": "<strong>Approve:</strong> Accept or reject the review<br/><strong>Validate:</strong> Review and continue the workflow<br/><strong>Resume Review:</strong> Review and continue the workflow",
```

Comment:
I think it would help to have this text a little bit more descriptive (maybe something like “Review the different components of a resume”). But it can be left as it.